config/redis.yml

```diff
@@ -102,6 +102,15 @@ development: &defaults
   saml_request_tracker:
     namespace: saml_request_tracker
     each_ttl: 3600 # 1 hour
+  iam_session:
+    namespace: iam-session
+    each_ttl: 1800
+  iam_user:
+    namespace: iam-user
+    each_ttl: 1800
+  iam_user_identity:
+    namespace: iam-user-identity
+    each_ttl: 1800
```

Comment:
Is this 30 minutes to match the access_token ttl?

Reply:
Yes, but this is a default. The TTL will be set from the introspect `exp` value.